### DISCUSSION TITLE
Delivery Data

### DISCUSSION THREAD

**Discussion by EH6451** [sdabhi23/bsedata#57] [Ideas]
Is there a way to get delivery data also ?

**Comment by sdabhi23** [comment]
It might be possible, I am not sure. Can you please point me to the relevant page on the bse website? (https://www.bseindia.com/ or https://m.bseindia.com/)

**Comment by EH6451** [comment]
hi,

https://www.bseindia.com/markets/equity/EQReports/GrossShortPos.aspx?flag=2

% deliverables quantity to traded quantity 

the table runs into several page/links
